elasticdl/python/tests/optimizer_wrapper_test.py

```diff
@@ -189,50 +189,6 @@ def test_allowed_slot_names(self):
         for opt, expected_slots in opt_and_slots_pairs:
             self._compare_slot_names(opt, expected_slots)
 
-    def _compare_initialize_values(self, opt, dim, slot, expected_init):
-        tmp = OptimizerWrapper(opt, None, {"test": dim})
-        self.assertTrue(
-            np.isclose(
-                tmp._initialize_unknown_slot("test", slot),
-                expected_init(dim).numpy(),
-            ).all()
-        )
-
-    def test_initialize(self):
```

Comment:
Delete test cases for `initialization` because new optimizer wrappers are not responsible for initialization.